File: internal/models/scenarios.go
// Copyright (c) 2024 Robert Cronin
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

package models

type Scenario struct {
	ID          int64    `json:"id"`
	Name        string   `json:"name"`
	Description string   `json:"description"`
	Tasks       []string `json:"tasks"`
	Validation  string   `json:"validation"`
}

File: app/models/scenarios.py
# Copyright (c) 2024 Robert Cronin
# 
# This software is released under the MIT License.
# https://opensource.org/licenses/MIT

from dataclasses import dataclass
from typing import List

@dataclass
class Scenario:
    id: int
    name: str
    description: str
    tasks: List[str]
    validation: str